manifest.xml

```diff
@@ -14,7 +14,7 @@
 		<Host Name="Workbook" />
 	</Hosts>
 	<DefaultSettings>
-		<SourceLocation DefaultValue="https://localhost:8081/index.html"/>
+		<SourceLocation DefaultValue="https://localhost:8080/index.html"/>
```

Comment:
Why the change in port?

Reply:
Because it is being served from the dev-server instead of the packager.